ArduCopter/Parameters.h

```diff
@@ -162,10 +162,11 @@ class Parameters {
         //
         // 80: Heli
         //
+        k_param_heli_servo_4 = 79,
         k_param_heli_servo_1 = 80,
         k_param_heli_servo_2,
         k_param_heli_servo_3,
-        k_param_heli_servo_4,
```

Comment:
The purpose is actually just that - in essence renaming the forth servo to servo_yaw and inserting a new servo 4. That way we won't loose any params when upgrading from older versions.